QUICKSTART-ECS.md

```diff
@@ -0,0 +1,191 @@
+# Using a Bottlerocket AMI with Amazon ECS
+
+> The [ECS variant](./aws-ecs.md) of Bottlerocket is currently in a developer preview phase and we're looking for your
+> [feedback](https://github.com/bottlerocket-os/bottlerocket#contact-us).
+> We'd love for you to try it out!
+
+[Amazon Elastic Container Service (Amazon ECS)](https://ecs.aws) is a highly scalable, fast, container management service that makes it easy to run, stop, and manage containers on a cluster.
```

Comment:
```suggestion
[Amazon Elastic Container Service (Amazon ECS)](https://ecs.aws) is a highly scalable container management service that makes it easy to run, stop, and manage containers on a cluster.
```
I think "fast" is implied enough, but if you love it, no comma after :)

Reply:
That sentence is taken verbatim from the first sentence [here](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/Welcome.html).  I think it's reasonable to keep them consistent, but if you think we should fix the comma in the developer guide as well we can do that too.

Reply:
I think those official docs need to be more verbose than we do, in general, and we can pare down a bit overall.

Reply:
Done